### DISCUSSION TITLE
deploy netlify button create an empty github repo

### DISCUSSION THREAD

**Discussion by makerjackie** [ObsidianPublisher/obsidian-github-publisher#122] [General]
see: https://github.com/Jackiexiao/publisher-template-netlify

ps: document https://obsidian-publisher.netlify.app/

it has two deploy button, which points to two different repo, is this right?

**Comment by Mara-Li** [comment]
Corrected in last documentation update.

**Reply by makerjackie** [reply]
sovled

**Comment by makerjackie** [comment]
it seems netlify sometimes won't clone repo, see: https://github.com/oleeskild/obsidian-digital-garden/issues/167#issuecomment-1399222123
